### DISCUSSION TITLE
Apply stretch through visualization parameter

### DISCUSSION THREAD

**Discussion by hillsonghimire** [opengeos/leafmap#668] [Q&A] [answered]
![image](https://github.com/opengeos/leafmap/assets/34327057/c6eeeaa3-151a-4448-8952-2642bb1375f4)
How can I set default stretch value from the visual parameter dictionary in leafmap. I couldn't find it any reference to that.

**Comment by giswqs** [comment]
This seems the interface of geemap, not leafmap.

**Comment by giswqs** [comment, marked as answer]
See https://geemap.org/workshops/AGU_2023/#visualizing-earth-engine-images
